src/software/simulation/simulator_robot.cpp

```diff
@@ -193,6 +207,8 @@ void SimulatorRobot::chip(float distance_m)
             float ground_velocity =
                 initial_velocity * static_cast<float>(chip_angle.cos());
             kick(ground_velocity);
+
+            dribbler_ball.can_be_kicked = false;
```

Comment:
Do we need this line? 

On line 180 we do `for(auto dribbler_ball : this->balls_in_dribbler_area)` which invokes the copy constructor of the ball (_I think_), so this change may never leave the loop.

Think you might need an `auto&`  instead of just `auto`

Reply:
Good catch. Fixed.